__tests__/news.test.js

```diff
@@ -0,0 +1,97 @@
+const app = require("../app.js");
+const request = require("supertest");
+const testData = require("../db/data/test-data/index.js");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+afterAll(() => {
+    return db.end();
+});
+
+describe("GET /api/topics", () => {
+
+    it("Should return an array of users objects", () => {
+
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then((response) => {
+                response.body.topics.forEach((topic) => {
+
+                    expect(topic).toHaveProperty("slug", expect.any(String));
+                    expect(topic).toHaveProperty("description", expect.any(String));
+
+                });
+            });
+    });
+
+    it("Should return the intended error message", () => {
+        return request(app)
+            .get("/api/banana")
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe("Page not found");
+            });
+    });
+
+    describe("GET /api/articles/:article_id", () => {
+
+        it.skip("Should return an array of articles objects", () => {
+            return request(app)
+                .get(`/api/articles/2`)
```

Comment:
(i realise i asked you to add this `.skip` but please make sure you remove it and fix that error before pushing again!)